python/tester/upload_hsc_rc2.py

```diff
@@ -187,9 +189,21 @@ def upload_hsc_images(group_id, butler, refs):
     refs : iterable of `lsst.daf.butler.DatasetRef`
         The datasets to upload
     """
+    try:
+        max_processes = math.ceil(0.25*multiprocessing.cpu_count())
```

Comment:
It was pretty arbitrary -- I didn't want to use a large fraction of shared resources, even if it would only be in bursts of ~15 seconds. On the current `rubin-devl`, `max_processes = 32`.